src/detectors/BEMC/ProtoCluster_factory_EcalBarrelSciGlassProtoClusters.h

```diff
@@ -33,6 +33,11 @@ class ProtoCluster_factory_EcalBarrelSciGlassProtoClusters : public JFactoryT<ed
         m_minClusterHitEdep=1.0 * dd4hep::MeV;    // from ATHENA reconstruction.py
         m_minClusterCenterEdep=30.0 * dd4hep::MeV; // from ATHENA reconstruction.py
 
+        // adjacency matrix
+        m_geoSvcName = "GeoSvc";
+        u_adjacencyMatrix = "";
```

Comment:
@veprbl Could you code-suggest what this should be and how I can see if it works here?

Reply:
```suggestion
        u_adjacencyMatrix = "(abs(row_1 - row_2) + abs(column_1 - column_2)) == 1";
```

Reply:
Oops, actually:
```suggestion
        u_adjacencyMatrix = "(abs(tower_1 - tower_2) + abs((sector_1 - sector2) * 5 + row_1 - row2)) == 1";
```